vtkVmtk/ComputationalGeometry/vtkvmtkSimplifyVoronoiDiagram.cxx

```diff
@@ -182,7 +187,18 @@ int vtkvmtkSimplifyVoronoiDiagram::RequestData(
   currentPolys->DeepCopy(inputPolys);
 
   currentLinks->Allocate(input->GetNumberOfPoints());
+#if VTK_MAJOR_VERSION >= 9 || (VTK_MAJOR_VERSION >= 8 && VTK_MINOR_VERSION >= 90)
```

Comment:
I am not tracking what this warning message is saying?
that `RequestData` needs to be re-written to comply with upstream vtk changes?

If `BuildLinks` doesn't exist anymore in VTK9 will then calls to `RequestData` fail?